### PR TITLE
chore: speed up kind.sh up a little 

### PR DESCRIPTION
**Pull request by chirino**
by installing all the operators in parallel and then wait their Readiness before installing Apex.

### CODE REVIEW

**Review by vishnoianil — APPROVED** [review]

**Comment by vishnoianil** [comment]
@chirino commits needs signoffs, so can you please signoff the commit.

**Comment by russellb** [comment]
lgtm, as well (once `Signed-off-by` gets added)

**Pull request merged**
